src/mbgl/map/tile_parser.cpp

```diff
@@ -117,7 +119,14 @@ std::unique_ptr<Bucket> TileParser::createBucket(const StyleBucket &bucketDesc)
         } else if (bucketDesc.type == StyleLayerType::Line) {
             return createLineBucket(*layer, bucketDesc);
         } else if (bucketDesc.type == StyleLayerType::Symbol) {
-            return createSymbolBucket(*layer, bucketDesc);
+            bool needsResources;
```

Comment:
Can we replace this with a `std::tuple` return value and a `std::tie` to expand the parameters?